src/User/Infrastructure/EventListener/AuthorizationRequestResolveListener.php

```
@@ -0,0 +1,41 @@
+        $token = $this->tokenStorage->getToken();
+        $user = $token?->getUser();
+
+        if ($user !== null) {
+            $event->resolveAuthorization(
+                AuthorizationRequestResolveEvent::AUTHORIZATION_APPROVED
+            );
+        } else {
+            $event->resolveAuthorization(
+                AuthorizationRequestResolveEvent::AUTHORIZATION_DENIED
+            );
+        }
```

PHPMD (2.15.0)

The method onAuthorizationRequestResolve uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)